test/python/tf_unittest_runner.py

```diff
@@ -53,7 +58,15 @@ def main():
         print('\n'.join(test_list))
     if (arguments.run_test):
         test_list = get_test_list(arguments.tensorflow_path, arguments.run_test)
+        print('\n'.join(test_list))
         run_test(test_list)
+    if (arguments.run_tests_from_file):
+        list_of_tests = run_tests_from_file(arguments.run_tests_from_file)
```

Comment:
the name of the function should be `read_tests_from_file` probably

Reply:
Hmm we are reading from file and running them instead of just using --run_test where you could input the test name. So run_tests_from_file makes more sense right?